lib/elixir/test/erlang/control_test.erl

```diff
@@ -14,8 +14,8 @@ cond_line_test() ->
 
 float_match_test() ->
   {'case', _, _,
-    [{clause, _, [{op, _, '+', {float, _, 0.0}}], [], [{atom, _, pos}]},
-     {clause, _, [{op, _, '-', {float, _, 0.0}}], [], [{atom, _, neg}]}]
+    [{clause, _, [{op, _, '+', {float, _, +0.0}}], [], [{atom, _, pos}]},
+     {clause, _, [{op, _, '-', {float, _, +0.0}}], [], [{atom, _, neg}]}]
```

Comment:
```suggestion
     {clause, _, [{op, _, '-', {float, _, -0.0}}], [], [{atom, _, neg}]}]
```

:D

Reply:
There is already the minus op, so that would make it a plus, unless I’m missing something?

Reply:
Oh doh, I was the one missing it, right!